consensus/misc/eip1559/eip1559.go

```diff
@@ -141,22 +182,26 @@ func CalcBaseFee(config *params.ChainConfig, parent *types.Header, time uint64)
 	}
 	elasticity := config.ElasticityMultiplier()
 	denominator := config.BaseFeeChangeDenominator(time)
-	if config.IsHolocene(parent.Time) {
+	var minBaseFeeLog2 uint8
+	if config.IsJovian(parent.Time) {
```

Comment:
should this be feature flagged?

Reply:
Yes. Here's how I did it for the calldata gas cost feature: https://github.com/niran/op-geth/commit/98e4a1fa3cc8f77f3b77d6a15573fe689e86ba20